benchmarks/runtime.bench.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { expect } = require('chai');
+const assert = require('assert');
```

Comment:
Should we use `node:assert` here?

Reply:
There are already a bunch of other node builtins being used, without the prefix. So I would make this change in a separate PR with a linting rule enforcing this.